packages/react-examples/src/react-experiments/UnifiedPeoplePicker/UnifiedPeoplePicker.WithEdit.Example.tsx

```diff
@@ -90,8 +90,46 @@ export const UnifiedPeoplePickerWithEditExample = (): JSX.Element => {
         />
       ),
     }),
+    itemComponent: ItemWithContextMenu<IPersona>({
```

Comment:
At this point, we don't know what items are selected, so I couldn't have two context menu options. Instead, the copy copies just that item or all selected depending on how many are selected. I think this functionality won't be too confusing to the user?

Reply:
I think that makes sense